src/main/kotlin/org/jmailen/gradle/kotlinter/pluginapplier/KotlinJvmSourceSetApplier.kt

```diff
@@ -11,7 +11,7 @@ import org.jmailen.gradle.kotlinter.id
 internal object KotlinJvmSourceSetApplier : SourceSetApplier {
 
     override fun applyToAll(project: Project, action: SourceSetAction) {
-        getSourceSets(project).all { sourceSet ->
+        getSourceSets(project).configureEach { sourceSet ->
```

Comment:
As a boy-scout rule, I replaced non-lazy apis usages with lazy ones

Reply:
Yes, in fact we had needed to be lazy in a few places in the code because if someone customizes the source sets in their build file we need to resolve our targets after that is configured, otherwise in early versions of kotlinter we didn't pick up the customization. I assume this is harmonious with that?

Reply:
AFAIR The old issue was about customization of an existing sourceset not being picked up. This change will "fix" the plugin if someone _registers new_ sourceset after kotlinter plugin gets applied. So yeah, it's definitelly consistent with the other behavior 👍